R/Connection.R

```diff
@@ -608,24 +608,94 @@ setMethod(
 
 #' List Available ODBC Drivers
 #'
-#' List the available drivers on your system. See the
-#' [Installation](https://github.com/r-dbi/odbc#installation) section of the
-#' package README for details on how to install drivers for the most common
-#' databases.
-#'
-#' @param keep A character vector of driver names to keep in the results, if
-#'   `NULL` (the default) will keep all drivers.
-#' @param filter A character vector of driver names to filter from the results, if
-#'   `NULL` (the default) will not filter any drivers.
+#' Collect information about the available driver names.
+#'
+#' @param keep A character vector of driver names to keep in the results.
```

Comment:
I'd suggest documenting `keep` and `filter` together since they're included mostly (I think) for some use by the IDE. Then you could also move the first sentence from the details here.

Reply:
[c99a80b](https://github.com/r-dbi/odbc/pull/649/commits/c99a80b2dd361ccecc5bbce11c63be7fc5f3e514).